### PR TITLE
Fix workflow for pinging maintainers in case of unlabeled PRs

### PR DESCRIPTION
**Pull request by pmeier**
Follow-up to #4012. I've tested all changes with PRs against this branch first. Thus, it should work now :crossed_fingers:

### CODE REVIEW

File: .github/process_commit.py [hunk 1]
```diff
@@ -33,46 +38,47 @@
...
-    else:
-        users = ()
-    return dict(
-        is_properly_labeled=is_properly_labeled,
-        responsible_users=", ".join(sorted([f"@{user}" for user in users])),
-    )
+    if is_properly_labeled:
+        return set()
+
+    return {merger, *get_pr_reviewers(pr_number)}
```
Comment (NicolasHug) [inline .github/process_commit.py:51]: nit: I'd just ping the merger, but it's not important

**Review by NicolasHug — APPROVED** [review]
Thanks @pmeier !

**Review by datumbox — APPROVED** [review]
@pmeier Thanks for the fix.

A future improvement might also be running this script only when the PR is against master. I noticed that all the PRs sent to your feature branch were untagged and the script kept pining you.

**Comment by pmeier** [comment]
@datumbox It does exactly that

https://github.com/pytorch/vision/blob/4db4d157d919d869888460f8473f33dfd6a30bfc/.github/workflows/pr-labels.yml#L3-L7

I only temporarily added this branch to the list for debug purposes.

File: .github/process_commit.py (hunk 1 above)
Comment (pmeier) [reply .github/process_commit.py:51]: Sometimes whoever merged it, did not dive deep into the PR, but rather relied on previous reviews. Note that not all reviewers, but only reviewers that approved the PR will get pinged:

https://github.com/pytorch/vision/blob/4db4d157d919d869888460f8473f33dfd6a30bfc/.github/process_pr.py#L72

**Comment by datumbox** [comment]
Didn't know you turned it off on purpose, very nice!

**Pull request merged**